ovos_core/intent_services/service.py

```
@@ -151,20 +152,24 @@ def disambiguate_lang(message):
         4 - config lang (or from message.data)
         """
         default_lang = get_message_lang(message)
-        valid_langs = get_valid_languages()
+        valid_langs = message.context.get("valid_langs") or get_valid_languages()
         valid_langs = [standardize_lang_tag(l) for l in valid_langs]
```

Ruff (0.11.9)

Ambiguous variable name: `l`

(E741)